README.md

```diff
@@ -783,6 +783,17 @@ Browsershot::url('https://example.com')
    ...
 ```
 
+#### Passing environment variables to the browser
+
+If you want to set custom environment variables which affect the browser instance you can use:
+
+```php
+Browsershot::url('https://example.com')
+   ->setOption('env.TZ', 'Pacific/Auckland')
```

Comment:
Rename this to `setEnvironmentOption`. 

Also add `setEnvironmentOptions` that accepts an array with options.